### ISSUE TITLE
ToDo: diffs FF79-FF80

### ISSUE THREAD

**Issue by earthlng** [ghacksuserjs/ghacks-user.js#994]
FF80 is [scheduled](https://wiki.mozilla.org/Release_Management/Calendar) for release August 25th

[FF80 release notes](https://www.mozilla.org/firefox/80.0/releasenotes/) [when ready]
[FF80 for developers](https://developer.mozilla.org/docs/Mozilla/Firefox/Releases/80)
[FF80 compatibility](https://www.fxsitecompat.dev/en-CA/versions/80/)
[FF80 security advisories](https://www.mozilla.org/security/advisories/mfsa2020-36/)

---

65 diffs ( 40 new, 8 gone, 17 different )

### new in v80.0:

* [x] pref("dom.security.https_only_mode_pbm", false); - https://github.com/ghacksuserjs/ghacks-user.js/commit/38d772e4c8721f59a361370572cdb25d777ea04a
* `0207` pref("browser.region.update.enabled", true); - actually added hidden in FF79 - bad earthlng 😀 
* FYI: pref("dom.block_download_insecure", false); [1614969](https://bugzilla.mozilla.org/show_bug.cgi?id=1614969)

### removed, renamed or hidden in  v80.0:
no changes necessary this time

### changed in  v80.0:
no changes necessary this time

---

### ignore

<details><summary>click me for details</summary><p>

==NEW
```js
pref("apz.force_disable_desktop_zooming_scrollbars", true);
pref("browser.preferences.experimental.hidden", false);
pref("browser.preferences.exposeHTTPSOnly", false);
pref("browser.tabs.documentchannel.parent-controlled", false);
pref("browser.topsites.useRemoteSetting", false);
pref("browser.urlbar.loglevel", "Error");
pref("browser.urlbar.update2", false);
pref("browser.urlbar.update2.localOneOffs", false);
pref("browser.urlbar.update2.oneOffsRefresh", false);
pref("browser.urlbar.update2.tabToComplete", false);
pref("devtools.netmonitor.audits.slow", 500);
pref("dom.security.https_only_mode_ever_enabled_pbm", false);
pref("dom.security.unexpected_system_load_telemetry_enabled", true);
pref("dom.suspend_inactive.enabled", false);
pref("domsecurity.httpsonly.infoURL", "https://developer.mozilla.org/en-US/docs/Glossary/https");
pref("extensions.backgroundServiceWorker.enabled", false);
pref("extensions.formautofill.addresses.capture.enabled", false);
pref("extensions.formautofill.creditCards.hideui", false);
pref("extensions.serviceWorkerRegister.allowed", false);
pref("fission.processOriginNames", false);
pref("gfx.use-ahardwarebuffer-content", false);
pref("layout.animation.prerender.partial.jank", false);
pref("layout.css.prefers-contrast.enabled", false);
pref("media.videocontrols.picture-in-picture.video-toggle.has-used", false);
pref("media.videocontrols.picture-in-picture.video-toggle.mode", -1);
pref("media.videocontrols.picture-in-picture.video-toggle.position", "right");
pref("print.tab_modal.enabled", false);
pref("security.bad_cert_domain_error.url_fix_enabled", false);
pref("security.ssl3.rsa_aes_128_gcm_sha256", true);
pref("security.ssl3.rsa_aes_256_gcm_sha384", true);
pref("security.warn_submit_secure_to_insecure", true);
pref("security.xfocsp.errorReporting.automatic", false);
pref("security.xfocsp.errorReporting.enabled", true);
pref("signon.management.page.fileImport.enabled", false);
pref("storage.sqlite.exclusiveLock.enabled", true);
pref("webgl.debug.incomplete-tex-color", 0);
pref("webgl.oop.via-pcq", false);
```

==REMOVED or HIDDEN
```js
pref("browser.urlbar.update2.expandTextOnFocus", false);
pref("devtools.accessibility.auto-init.enabled", true);
pref("devtools.responsive.browserUI.enabled", true);
pref("dom.sidebar.enabled", false);
pref("layout.viewport_contains_no_contents_area", true);
pref("media.getusermedia.experimental_input_processing", false);
pref("network.standard-url.punycode-host", true);
pref("webgl.prototype.ipc-pcq", 0);
```

==CHANGED
```js
pref("apz.mvm.force-enabled", true); // prev: false
pref("browser.aboutwelcome.overrideContent", "{\"id\": \"multi-stage-welcome-default\",\"template\": \"multistage\",\"screens\": [{\"id\": \"AW_GET_STARTED\",\"order\": 0,\"content\": {\"zap\": true,\"title\": {\"string_id\": \"onboarding-multistage-welcome-header\"},\"subtitle\": {\"string_id\": \"onboarding-multistage-welcome-subtitle\"},\"primary_button\": {\"label\": {\"string_id\": \"onboarding-multistage-welcome-primary-button-label\"},\"action\": {\"navigate\": true}},\"secondary_button\": {\"text\": {\"string_id\": \"onboarding-multistage-welcome-secondary-button-text\"},\"label\": {\"string_id\": \"onboarding-multistage-welcome-secondary-button-label\"},\"position\": \"top\",\"action\": {\"type\": \"OPEN_URL\",\"addFlowParams\": true,\"data\": {\"args\": \"https://accounts.firefox.com/?service=sync&action=email&context=fx_desktop_v3&entrypoint=activity-stream-firstrun&style=trailhead\",\"where\": \"current\"}}}}}, {\"id\": \"AW_IMPORT_SETTINGS\",\"order\": 1,\"content\": {\"zap\": true, \"disclaimer\": {\"string_id\": \"onboarding-import-sites-disclaimer\"},\"title\": {\"string_id\": \"onboarding-multistage-import-header\"},\"subtitle\": {\"string_id\": \"onboarding-multistage-import-subtitle\"},\"tiles\": {\"type\": \"topsites\",\"info\": true},\"primary_button\": {\"label\": {\"string_id\": \"onboarding-multistage-import-primary-button-label\"},\"action\": {\"type\": \"SHOW_MIGRATION_WIZARD\",\"navigate\": true}},\"secondary_button\": {\"label\":  {\"string_id\": \"onboarding-multistage-import-secondary-button-label\"},\"action\": {\"navigate\": true}}}}, {\"id\": \"AW_CHOOSE_THEME\",\"order\": 2,\"content\": {\"zap\": true,\"title\":  {\"string_id\": \"onboarding-multistage-theme-header\"},\"subtitle\": {\"string_id\": \"onboarding-multistage-theme-subtitle\"},\"tiles\": {\"type\": \"theme\",\"action\": {\"theme\": \"<event>\"}, \"data\": [{\"theme\": \"automatic\",\"label\": {\"string_id\": \"onboarding-multistage-theme-label-automatic\"}, \"description\": {\"string_id\": \"onboarding-multistage-theme-description-automatic\"}, \"tooltip\": {\"string_id\": \"onboarding-multistage-theme-tooltip-automatic\"}}, {\"theme\": \"light\",\"label\": {\"string_id\": \"onboarding-multistage-theme-label-light\"}, \"tooltip\": {\"string_id\": \"onboarding-multistage-theme-tooltip-light\"}},{\"theme\": \"dark\",\"label\": {\"string_id\": \"onboarding-multistage-theme-label-dark\"}, \"tooltip\": {\"string_id\": \"onboarding-multistage-theme-tooltip-dark\"}}]},\"primary_button\": {\"label\": {\"string_id\": \"onboarding-multistage-theme-primary-button-label\"},\"action\": {\"navigate\": true}},\"secondary_button\": {\"label\": {\"string_id\": \"onboarding-multistage-theme-secondary-button-label\"},\"action\": {\"theme\": \"automatic\",\"navigate\": true}}}}]}"); // prev: ""
pref("browser.newtabpage.activity-stream.discoverystream.personalization.version", 2); // prev: 1
pref("browser.newtabpage.activity-stream.discoverystream.region-layout-config", "US,CA,GB,DE,IE"); // prev: "US,CA,GB,DE"
pref("browser.newtabpage.activity-stream.discoverystream.region-stories-config", "US,DE,CA,GB,IE"); // prev: "US,DE,CA,GB"
pref("dom.animations-api.compositing.enabled", true); // prev: false
pref("dom.arena_allocator.enabled", true); // prev: false
pref("dom.max_chrome_script_run_time", 0); // prev: 20
pref("extensions.blocklist.useMLBF", true); // prev: false
pref("extensions.blocklist.useMLBF.stashes", true); // prev: false
pref("identity.sync.useOAuthForSyncToken", true); // prev: false
pref("media.navigator.video.use_transport_cc", true); // prev: false
pref("media.peerconnection.video.use_rtx", true); // prev: false
pref("media.peerconnection.video.use_rtx.blocklist", ""); // prev: "*.google.com"
pref("security.allow_disjointed_external_uri_loads", false); // prev: true
pref("signon.backup.enabled", true); // prev: false
pref("signon.capture.inputChanges.enabled", false); // prev: true
```

</p></details>

**Comment by earthlng** [comment]
<details><summary>some bugzilla tickets</summary><p>

* apz.force_disable_desktop_zooming_scrollbars
Bug [1655160](https://bugzilla.mozilla.org/show_bug.cgi?id=1655160). Disable new desktop zooming scrollbar code for now.
Bug [1651332](https://bugzilla.mozilla.org/show_bug.cgi?id=1651332). Create a pref to gate the new scrollbar code on.

* apz.mvm.force-enabled
Bug [1648687](https://bugzilla.mozilla.org/show_bug.cgi?id=1648687) - Disable the visual-viewport-only MVM for Firefox 79.
Bug [1644271](https://bugzilla.mozilla.org/show_bug.cgi?id=1644271) - Enable the MVM pref by default.
Bug [1644271](https://bugzilla.mozilla.org/show_bug.cgi?id=1644271) - Add a pref to enable the MobileViewportManager.

* browser.aboutwelcome.overrideContent
Bug [1656631](https://bugzilla.mozilla.org/show_bug.cgi?id=1656631) - Make static sites in import screen as default experience.
Bug [1653322](https://bugzilla.mozilla.org/show_bug.cgi?id=1653322) - Converted import topsites &quot;disclaimer&quot; tooltip to a footer
Bug [1653087](https://bugzilla.mozilla.org/show_bug.cgi?id=1653087) - Add default theme as an option to multistage choose a theme screen
Bug [1649292](https://bugzilla.mozilla.org/show_bug.cgi?id=1649292) - Make multistage about:welcome as default UX
Bug [1637079](https://bugzilla.mozilla.org/show_bug.cgi?id=1637079) - Initial multi stage about:welcome layout

* browser.newtabpage.activity-stream.discoverystream.personalization.version
Bug [1644890](https://bugzilla.mozilla.org/show_bug.cgi?id=1644890) - newtab stories v2 targeting on by default
Bug [1644889](https://bugzilla.mozilla.org/show_bug.cgi?id=1644889) - Turn on newtab story targeting v2 in nightly.

* browser.newtabpage.activity-stream.discoverystream.region-layout-config
Bug [1657530](https://bugzilla.mozilla.org/show_bug.cgi?id=1657530) - Enable Ireland region for newtab stories.
Bug [1653061](https://bugzilla.mozilla.org/show_bug.cgi?id=1653061) - turn on 7 rows of pocket stories in the de locale
Bug [1653061](https://bugzilla.mozilla.org/show_bug.cgi?id=1653061) - turn on 7 rows of pocket stories in the de locale.
Bug [1632574](https://bugzilla.mozilla.org/show_bug.cgi?id=1632574) - Turn Pocket stories on by default in GB
Bug [1632574](https://bugzilla.mozilla.org/show_bug.cgi?id=1632574) - Turn Pocket stories on by default in GB.

* browser.newtabpage.activity-stream.discoverystream.region-stories-config
Bug [1657530](https://bugzilla.mozilla.org/show_bug.cgi?id=1657530) - Enable Ireland region for newtab stories.
Bug [1632574](https://bugzilla.mozilla.org/show_bug.cgi?id=1632574) - Turn Pocket stories on by default in GB
Bug [1632574](https://bugzilla.mozilla.org/show_bug.cgi?id=1632574) - Turn Pocket stories on by default in GB.

* browser.preferences.experimental.hidden
Bug [1648222](https://bugzilla.mozilla.org/show_bug.cgi?id=1648222) - Hide the experimental features category if there are no available experimental features.

* browser.preferences.exposeHTTPSOnly
Bug [1644152](https://bugzilla.mozilla.org/show_bug.cgi?id=1644152): Expose HTTPS-Only Mode in about:preferences.

* browser.region.update.enabled
Bug [1651124](https://bugzilla.mozilla.org/show_bug.cgi?id=1651124) - Enable region cachebusting on release
Bug [1627555](https://bugzilla.mozilla.org/show_bug.cgi?id=1627555) - Update region if in new location for a length of time.

* browser.tabs.documentchannel.parent-controlled
Bug [1647557](https://bugzilla.mozilla.org/show_bug.cgi?id=1647557) - Add preffed-off code for controlling document loads directly from CanonicalBrowsingContext.

* browser.topsites.useRemoteSetting
Bug [1653932](https://bugzilla.mozilla.org/show_bug.cgi?id=1653932) - Add pref for top site defaults from remote settings.

* browser.urlbar.loglevel
Bug [1650099](https://bugzilla.mozilla.org/show_bug.cgi?id=1650099) - Unify UrlbarProviders logger boilerplate.

* browser.urlbar.update2
Bug [1647930](https://bugzilla.mozilla.org/show_bug.cgi?id=1647930) - Create urlbar.update2 prefs.

* browser.urlbar.update2.expandTextOnFocus
Bug [1647930](https://bugzilla.mozilla.org/show_bug.cgi?id=1647930) - Create urlbar.update2 prefs.

* browser.urlbar.update2.localOneOffs
Bug [1647930](https://bugzilla.mozilla.org/show_bug.cgi?id=1647930) - Create urlbar.update2 prefs.

* browser.urlbar.update2.oneOffsRefresh
Bug [1647930](https://bugzilla.mozilla.org/show_bug.cgi?id=1647930) - Create urlbar.update2 prefs.

* browser.urlbar.update2.tabToComplete
Bug [1647930](https://bugzilla.mozilla.org/show_bug.cgi?id=1647930) - Create urlbar.update2 prefs.

* devtools.accessibility.auto-init.enabled
Bug [1640778](https://bugzilla.mozilla.org/show_bug.cgi?id=1640778) - remove auto-init experiment code.
Bug [1642520](https://bugzilla.mozilla.org/show_bug.cgi?id=1642520) - enable accessibility panel auto initialization on all channels.
Bug [1602075](https://bugzilla.mozilla.org/show_bug.cgi?id=1602075) - add an accessibility-panel-auto-init feature to control the panel auto enabling functionality.

* devtools.netmonitor.audits.slow
Bug [1648373](https://bugzilla.mozilla.org/show_bug.cgi?id=1648373) - Add slow requests indicator

* devtools.responsive.browserUI.enabled
Bug [1585097](https://bugzilla.mozilla.org/show_bug.cgi?id=1585097) - Delete the devtools.responsive.browserUI.enabled pref.
Bug [1585005](https://bugzilla.mozilla.org/show_bug.cgi?id=1585005)  - Enable RDM Fission for all builds.
Bug [1621306](https://bugzilla.mozilla.org/show_bug.cgi?id=1621306) - Enable the devtools.responsive.browserUI.enabled pref in Nightly only

* dom.animations-api.compositing.enabled
Bug [1652676](https://bugzilla.mozilla.org/show_bug.cgi?id=1652676) - Enable Web Animations API compositing operations on release channels;

* dom.arena_allocator.enabled
Bug [1652496](https://bugzilla.mozilla.org/show_bug.cgi?id=1652496) - Enable DOM arena allocation in Release

* dom.block_download_insecure
Bug [1614969](https://bugzilla.mozilla.org/show_bug.cgi?id=1614969) - Check download with MixedContentBlocker

* dom.max_chrome_script_run_time
Bug [1654325](https://bugzilla.mozilla.org/show_bug.cgi?id=1654325) - disable the parent process slow script dialog,

* dom.security.https_only_mode_ever_enabled_pbm
Bug [1647719](https://bugzilla.mozilla.org/show_bug.cgi?id=1647719): Introduce Pref for HTTS-Only in Private Browsing Mode.

* dom.security.https_only_mode_pbm
Bug [1647719](https://bugzilla.mozilla.org/show_bug.cgi?id=1647719): Introduce Pref for HTTS-Only in Private Browsing Mode.

* dom.security.unexpected_system_load_telemetry_enabled
Bug [1644671](https://bugzilla.mozilla.org/show_bug.cgi?id=1644671) - systemprincipal restrictions telemetry

* dom.sidebar.enabled
Bug [1632448](https://bugzilla.mozilla.org/show_bug.cgi?id=1632448) - Remove AddSearchProvider implementation.

* dom.suspend_inactive.enabled
Bug [1639577](https://bugzilla.mozilla.org/show_bug.cgi?id=1639577) - Only suspend GeckoView tabs that are alone in their BrowsingContextGroup

* domsecurity.httpsonly.infoURL
Bug [1644152](https://bugzilla.mozilla.org/show_bug.cgi?id=1644152): Expose HTTPS-Only Mode in about:preferences.

* extensions.backgroundServiceWorker.enabled
Bug [1609920](https://bugzilla.mozilla.org/show_bug.cgi?id=1609920) - part 1: Allow the WebExtension Framework to register a moz-extension service worker.

* extensions.blocklist.useMLBF
Bug [1620621](https://bugzilla.mozilla.org/show_bug.cgi?id=1620621) - Implement blocklist stashing behind pref
Bug [1620621](https://bugzilla.mozilla.org/show_bug.cgi?id=1620621) - Add bloomfilter-based blocklist for addons
Bug [1631018](https://bugzilla.mozilla.org/show_bug.cgi?id=1631018) - Enable blocklist v3 by default on desktop

* extensions.blocklist.useMLBF.stashes
Bug [1620621](https://bugzilla.mozilla.org/show_bug.cgi?id=1620621) - Implement blocklist stashing behind pref
Bug [1631018](https://bugzilla.mozilla.org/show_bug.cgi?id=1631018) - Enable blocklist v3 by default on desktop

* extensions.formautofill.addresses.capture.enabled
Bug [1626055](https://bugzilla.mozilla.org/show_bug.cgi?id=1626055) - Disable address autofill auto-saving until we deal with duplicate addresses.

* extensions.formautofill.creditCards.hideui
Bug [1658027](https://bugzilla.mozilla.org/show_bug.cgi?id=1658027) - Separate visual interactions on Credit Card Autofill for A/B testing.

* extensions.serviceWorkerRegister.allowed
Bug [1609920](https://bugzilla.mozilla.org/show_bug.cgi?id=1609920) - part 2: Guard ServiceWorkerContainer::Register to allow/disallow moz-extension scheme based on prefs.

* fission.processOriginNames
Bug [1642772](https://bugzilla.mozilla.org/show_bug.cgi?id=1642772): Add fission process renaming to eTLD+1 for Nightly (only) via a pref

* gfx.use-ahardwarebuffer-content
Bug [1648411](https://bugzilla.mozilla.org/show_bug.cgi?id=1648411) - Add AHardwareBuffer layer buffer support on android

* identity.sync.useOAuthForSyncToken
Bug [1648194](https://bugzilla.mozilla.org/show_bug.cgi?id=1648194) - Enable 'identity.sync.useOAuthForSyncToken' for all users.
Bug [1635937](https://bugzilla.mozilla.org/show_bug.cgi?id=1635937) - Enable 'identity.sync.useOAuthForSyncToken' pref for Nightly users
Bug [1631830](https://bugzilla.mozilla.org/show_bug.cgi?id=1631830) - Fetch Sync tokens with OAuth behind a pref

* layout.animation.prerender.partial.jank
Bug [1324591](https://bugzilla.mozilla.org/show_bug.cgi?id=1324591) - Report janked animations to the main-thread and update them on the main-thread.

* layout.css.prefers-contrast.enabled
Bug [1506364](https://bugzilla.mozilla.org/show_bug.cgi?id=1506364) - Implement the prefers-contrast media-query.

* layout.viewport_contains_no_contents_area
Bug [1650686](https://bugzilla.mozilla.org/show_bug.cgi?id=1650686) - Drop layout.viewport_contains_no_contents_area.

* media.getusermedia.experimental_input_processing
Bug [1646660](https://bugzilla.mozilla.org/show_bug.cgi?id=1646660) - Disable cheaper processing made in bug [1628779](https://bugzilla.mozilla.org/show_bug.cgi?id=1628779).
Bug [1628779](https://bugzilla.mozilla.org/show_bug.cgi?id=1628779) - Pass the audio rountrip latency to the echo canceller, and disable extended filter and delay agnostic AEC, on macOS Nightly.

* media.navigator.video.use_transport_cc
Bug [1651722](https://bugzilla.mozilla.org/show_bug.cgi?id=1651722) - Enable transport-cc and rtx and let them ride the trains;
Bug [1625803](https://bugzilla.mozilla.org/show_bug.cgi?id=1625803) - Enable transport-cc extension for nightly and early beta;
Bug [1606823](https://bugzilla.mozilla.org/show_bug.cgi?id=1606823) - Add support for WebRTC transport-cc extension.

* media.peerconnection.video.use_rtx
Bug [1651722](https://bugzilla.mozilla.org/show_bug.cgi?id=1651722) - Enable transport-cc and rtx and let them ride the trains;
Bug [1641600](https://bugzilla.mozilla.org/show_bug.cgi?id=1641600) - Re-enable rtx and add *.google.com to blocklist;
Bug [1638758](https://bugzilla.mozilla.org/show_bug.cgi?id=1638758) - Turn off just-added media.peerconnection.video.use_rtx in nightly.
Bug [1632489](https://bugzilla.mozilla.org/show_bug.cgi?id=1632489) - Enable rtx for early beta or earlier;
acked out 4 changesets (bug [1632489](https://bugzilla.mozilla.org/show_bug.cgi?id=1632489)) for android crashes at test_peerConnection_maxFsConstraint.html. CLOSED TREE
Bug [1164187](https://bugzilla.mozilla.org/show_bug.cgi?id=1164187) - Add pref for rtx;

* media.peerconnection.video.use_rtx.blocklist
Bug [1651722](https://bugzilla.mozilla.org/show_bug.cgi?id=1651722) - Enable transport-cc and rtx and let them ride the trains;
Bug [1641600](https://bugzilla.mozilla.org/show_bug.cgi?id=1641600) - Re-enable rtx and add *.google.com to blocklist;

* media.videocontrols.picture-in-picture.video-toggle.has-used
Bug [1653663](https://bugzilla.mozilla.org/show_bug.cgi?id=1653663) - Add experimental Picture-in-Picture toggle variations.

* media.videocontrols.picture-in-picture.video-toggle.mode
Bug [1653663](https://bugzilla.mozilla.org/show_bug.cgi?id=1653663) - Add experimental Picture-in-Picture toggle variations.

* media.videocontrols.picture-in-picture.video-toggle.position
Bug [1653663](https://bugzilla.mozilla.org/show_bug.cgi?id=1653663) - Add experimental Picture-in-Picture toggle variations.

* network.standard-url.punycode-host
Bug [1637727](https://bugzilla.mozilla.org/show_bug.cgi?id=1637727) - convert network.standard-url.punycode-host to a StaticPref.
Bug [1653201](https://bugzilla.mozilla.org/show_bug.cgi?id=1653201) - Remove internal nsStandardURL punycode pref

* print.tab_modal.enabled
Bug [1652627](https://bugzilla.mozilla.org/show_bug.cgi?id=1652627) - Setup a pref for tab modal print UI

* security.allow_disjointed_external_uri_loads
Bug [1650162](https://bugzilla.mozilla.org/show_bug.cgi?id=1650162) - Turn security.allow_disjointed_external_uri_loads back on to fix regressions opening external applications.
Bug [1606797](https://bugzilla.mozilla.org/show_bug.cgi?id=1606797) - do not allow navigating to external URIs in cross-origin disjoint browsing contexts,

* security.bad_cert_domain_error.url_fix_enabled
Bug [1656253](https://bugzilla.mozilla.org/show_bug.cgi?id=1656253) - disable security.bad_cert_domain_error.url_fix_enabled in late beta and release.
Bug [1617987](https://bugzilla.mozilla.org/show_bug.cgi?id=1617987) - Fix URLs by prefixing www. when users encounter bad cert domain errors.

* security.ssl3.rsa_aes_128_gcm_sha256
Bug [1641356](https://bugzilla.mozilla.org/show_bug.cgi?id=1641356) - Unhide RSA-AES-GCM ciphersuite preferences

* security.ssl3.rsa_aes_256_gcm_sha384
Bug [1641356](https://bugzilla.mozilla.org/show_bug.cgi?id=1641356) - Unhide RSA-AES-GCM ciphersuite preferences

* security.warn_submit_secure_to_insecure
Bug [436200](https://bugzilla.mozilla.org/show_bug.cgi?id=436200) - Add a pref to turn off warning submitting secure to insecure.

* security.xfocsp.errorReporting.automatic
Bug [1647825](https://bugzilla.mozilla.org/show_bug.cgi?id=1647825) - Part 1: Add the UI for enable reporting XFO and CSP:frame-ancestors error.

* security.xfocsp.errorReporting.enabled
Bug [1647825](https://bugzilla.mozilla.org/show_bug.cgi?id=1647825) - Part 1: Add the UI for enable reporting XFO and CSP:frame-ancestors error.

* signon.backup.enabled
Bug [1593467](https://bugzilla.mozilla.org/show_bug.cgi?id=1593467) - Automatically restore from logins-backup.json when logins.json is missing or corrupt.
Bug [1597358](https://bugzilla.mozilla.org/show_bug.cgi?id=1597358) - Create a backup of logins.json to use when logins.json is missing or corrupt.

* signon.capture.inputChanges.enabled
Bug [1654405](https://bugzilla.mozilla.org/show_bug.cgi?id=1654405) - pref doorhanger v2 off outside Nightly;r=MattN
Bug [1641412](https://bugzilla.mozilla.org/show_bug.cgi?id=1641412) - Pass all possible UNs/PWs to doorhanger;r=MattN,sfoster

* signon.management.page.fileImport.enabled
Bug [1654604](https://bugzilla.mozilla.org/show_bug.cgi?id=1654604) - Hold CSV Import to Nightly for Fx80.

* storage.sqlite.exclusiveLock.enabled
Bug [1650201](https://bugzilla.mozilla.org/show_bug.cgi?id=1650201) - Fix mozStorage prefs read before profile and fallback to a non-exclusive VFS when it can't get an exclusive lock.

* webgl.debug.incomplete-tex-color
Bug [1649894](https://bugzilla.mozilla.org/show_bug.cgi?id=1649894) - Add webgl.debug.incomplete-tex-color.

* webgl.oop.via-pcq
Bug [1607940](https://bugzilla.mozilla.org/show_bug.cgi?id=1607940) - Stand up webgl.out-of-process:true path.

* webgl.prototype.ipc-pcq
Bug [1607940](https://bugzilla.mozilla.org/show_bug.cgi?id=1607940) - Stand up webgl.out-of-process:true path.

</p></details>

**Comment by earthlng** [comment]
`browser.region.update.enabled` - https://github.com/ghacksuserjs/ghacks-user.js/issues/978#issuecomment-665589158

re: 1244 suggestions - the SETTING line already has `(FF81+)`. Assuming they'll flip `exposeHTTPSOnly` in FF81, I think that's fine.

Yes we can add `https_only_mode_pbm`. The one we already have, `https_only_mode`, enables HOM for both normal and private windows, whereas the new one enables it for private windows only.
If `https_only_mode` is **true**, `https_only_mode_pbm` is ignored

**Comment by geeknik** [comment]
[browser.region.update.enabled](https://github.com/ghacksuserjs/ghacks-user.js/issues/978#issuecomment-671459322) is set to true for FF80 as per this [commit](https://hg.mozilla.org/mozilla-central/rev/2bbf5b2b98a3f0edf4efe5b14c2428e0c5afb90e). [Relevant bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1651124).

**Comment by earthlng** [comment]
`security.xfocsp.errorReporting.*` - https://bugzilla.mozilla.org/show_bug.cgi?id=1647825#c9:

> The reporting is disabled by default. When users hit an XFO or a CSP error page, there is a checkbox UI to allow user opt-in the reporting. Once users check the checkbox, we will start to report. So, we will report every time users hit an XFO or CSP error page. And users can opt-out the reporting by unchecking the checkbox. After that, we will no longer report the error.

screenshot of the error page with the checkbox: https://bug1647825.bmoattachments.org/attachment.cgi?id=9162074

"The reporting is disabled by default" = `security.xfocsp.errorReporting.automatic=false`

Checking the checkbox flips the .automatic pref to true.
Setting `security.xfocsp.errorReporting.enabled` to `false` hides the checkbox.

Uses normal telemetry ping (which we disable) and ["The client_id and environment are not sent with this ping."](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/data/xfocsp-error-report-ping.html)

So in our case the checkbox does nothing and we could include these 2 prefs to (1) make sure the reporting is disabled and (2) the checkbox is not visible. Or just move them to ignore

**Comment by earthlng** [comment]
`security.allow_disjointed_external_uri_loads` - caused some regressions initially but they apparently think it's ready now after landing some fixes in FF80.

```
# Whether window A is allowed to navigate cross-origin window B (that is not
# a descendant frame of A) to a URI that loads externally.
```
[source](https://hg.mozilla.org/mozilla-central/rev/47c229951c8b#l1.12)

"allow...=false" is what we want. move to ignore IMO

**Comment by Thorin-Oakenpants** [comment]
> Assuming they'll flip `exposeHTTPSOnly` in FF81, I think that's fine.

I doubt it will land in FF81 .. it's not even going to be dev/beta 81 (99% sure, but I'll wait for dev/beta 81 to land to comfirm) - there are too many blockers IMO. 

My thinking **now** is we just remove it and wait for the pref to get flipped (or removed) in stable and that's out reminder to add the `[setting]` tag

**Comment by Thorin-Oakenpants** [comment]
I did a PR .. I don't mind what I did there, or we can rip it out

**Comment by Thorin-Oakenpants** [comment]
> `security.xfocsp.errorReporting.*` ... So in our case the checkbox does nothing and we could include these 2 prefs to (1) make sure the reporting is disabled and (2) the checkbox is not visible. Or just move them to ignore

I'm 51/49 in favor of ignoring ... I typed out the discussion in my head and got nowhere: tl;dr you and I are fine, and we will always pick up if the automatic pref is flipped, and the less stuff in the user.js the better. That said, defence in depth is good, especially since URIs would be recorded and that data would be kept (I think: I'm not sure what happens when the pings fails - are you talking about `browser.ping-centre.telemetry` in `0506`?)

The choice is yours @earthlng

**Comment by earthlng** [comment]
> I'm 51/49 in favor of ignoring

I agree. It's opt-in and the worst that could happen is someone willingly opting-in to help Mozilla identify and block malicious sites and ending up not sending anything because we have all telemetry disabled.
Even if it would send a ping somehow, "the client_id and environment are not sent with this ping." and thus the only potentially linkable/identifiable "id" would be the IP and IDK if they record/store that. But even if they do record and/or store that - it's opt-in!

**Comment by Thorin-Oakenpants** [comment]
[source](https://gitlab.torproject.org/tpo/applications/tor-browser/-/issues/40095#note_2707260)

> https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Releases/80 unexciting (which is actually a feature)

LOL

---

what's left
- A; new: pref("dom.block_download_insecure", false);
- B: new: pref("dom.security.unexpected_system_load_telemetry_enabled", true);
- C: changed: pref("signon.backup.enabled", true); // prev: false
- D: changed: pref("signon.capture.inputChanges.enabled", false); // prev: true

C: ignore IMO: a local backed up passwords file is (probably) no more dangerous than having the original. I haven't read anything about it, so I don't know if it keeps multiple copies or how often or if it's triggered when you add/remove passwords. But it can't hurt for end users, except that if you removed the account details for `secretplace.com`, there could still be a record of it. We're getting a bit edge case here.

D: NFI : feel free to look up if it's harmless

A: this has some regressions. I wouldn't expect any of our users to get HTTP downloads on secure pages. We could either wait for it to get flipped, or add it now (**because who knows if it will take 14 releases to get flipped like `noopener`**)

B:E said : should be covered by disabling telemetry but there's an [unfixed regression causing a crash](https://bugzilla.mozilla.org/show_bug.cgi?id=1657208), so maybe we should disable this
  - IMO ignore it: if crashes were that bad, Moz would do a point release

tl;dr: B,C,D = ignore; A = ignore and make a note (new issue as reminder) to revisit in 3 or 4 releases

@earthlng and co, wot say you

**Comment by geeknik** [comment]
D: [commit](https://hg.mozilla.org/mozilla-central/rev/85bbe760e98ea9edc1d6ad8550359e0274c0d00d) - [Bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1641412).  ¯\\\_(ツ)_/¯

**Comment by Thorin-Oakenpants** [comment]
^^ Thanks. Sorry for my laziness .. I know the relevant bugzillas are listed by earthlng: and I see most bugs because every day or so I check out all landed bugs as I look for possible version feature detection items.

D: Ahhh, yeah, we can ignore that

**Comment by rusty-snake** [comment]
> I wouldn't expect any of our users to get HTTP downloads on secure pages.

HTTP downloads are still common if mirrors are used because of big downloads.

This pref is on about:config but not in dxr. :rofl:  LOL.

**Comment by Thorin-Oakenpants** [comment]
I linked the ticket above, and I have read of regressions (like downloads folder has wrong info, no notifications etc) of which some/all are now fixed .. so I was just going to ignore it for now and see if it was flipped on in a couple of releases

I was trying to find a meta ticket, but I didn't try very hard

**Comment by Thorin-Oakenpants** [comment]
Here's a bit of a dig around

Depends on
- [1654780](https://bugzilla.mozilla.org/show_bug.cgi?id=1654780) drag and drop bypasses

Regressions
- open [1654139](https://bugzilla.mozilla.org/show_bug.cgi?id=1654139) silent failure to DL
- fixed 82: [1656296](https://bugzilla.mozilla.org/show_bug.cgi?id=1656296) = add indicator for blocked downloads
- open [1660969](https://bugzilla.mozilla.org/show_bug.cgi?id=1660969) ship panel/button to manually confirm/override block
- open [1660952](https://bugzilla.mozilla.org/show_bug.cgi?id=1660952) probable duplicate or may be windows only

Lots of others marked as duplicates
- pdfs not loading (if not viewed internally)
- other DLs nothing happens or insecure warnings

One of the comments in one of those tickets says telemetry indicates **about 10% of downloads initiated from secure sites are insecure**. So I guess they just need to get user notification and manual overrides working, and the we can flip it on
